lib/global-variable-adapter.js

```diff
@@ -0,0 +1,27 @@
+var createLocalStorageAdapter = require('./local-storage-adapter')
+
+module.exports = createGlobalVariableAdapter
+
+function createGlobalVariableAdapter (queueName) {
+  window.__queueThat__ = window.__queueThat__ || {}
+
+  var localStorageAdapter = createLocalStorageAdapter(queueName)
```

Comment:
I guess I find this a little weird, but I get what is going on. I think I initially expected there would be entirely separate `localStorage` vs `variable` adapters, however this is just patching the local storage adapter to use a variable so we can reuse it's other functionality.